eng/stylecop.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+    "settings": {        
+        "indentation": {
+            "indentationSize": 4,
+            "tabSize": 4,
+            "useTabs": false
+        },
+        "orderingRules": {
+            "usingDirectivesPlacement": "outsideNamespace",
+            "blankLinesBetweenUsingGroups": "omit"
+        },
+        "namingRules": {
+            "allowCommonHungarianPrefixes": true,
```

Comment:
Can you explain why is this and the next one needed? Why Hungarian in particular?

Reply:
I took this from a template and forgot to check. These two sections are unused here.